symfony/plugins/orangehrmPimPlugin/Api/EmpEmergencyContactAPI.php

```diff
@@ -0,0 +1,313 @@
+                new Rule(Rules::IN_ACCESSIBLE_EMP_NUMBERS)
+            ),
+            ...$this->getCommonBodyValidationRules(),
+        );
+    }
+
+    private function getCommonBodyValidationRules(): array
+    {
+        return [
+            new ParamRule(self::PARAMETER_NAME,
+                new Rule(Rules::STRING_TYPE),
+                new Rule(Rules::LENGTH, [null, self::PARAM_RULE_Max_Length])
+            ),
+            new ParamRule(self::PARAMETER_RELATIONSHIP,
+                new Rule(Rules::STRING_TYPE),
+                new Rule(Rules::LENGTH, [null, self::PARAM_RULE_Max_Length])
+            ),
+            new ParamRule(self::PARAMETER_HOME_PHONE,
+                new Rule(Rules::STRING_TYPE),
+                new Rule(Rules::LENGTH, [null, self::PARAM_RULE_Max_Length])
+            ),
+            new ParamRule(self::PARAMETER_OFFICE_PHONE,
+                new Rule(Rules::STRING_TYPE),
+                new Rule(Rules::LENGTH, [null, self::PARAM_RULE_Max_Length])
+            ),
+            new ParamRule(self::PARAMETER_MOBILE_PHONE,
+                new Rule(Rules::STRING_TYPE),
+                new Rule(Rules::LENGTH, [null, self::PARAM_RULE_Max_Length])
+            ),
+        ];
+    }
+
+    /**
+     * @inheritDoc
+     * @throws DaoException
+     */
+    public function delete(): EndpointDeleteResult
+    {
+        $sequenceNumbers = $this->getRequestParams()->getArray(RequestParams::PARAM_TYPE_BODY,
+            CommonParams::PARAMETER_IDS);
+        $empNumber = $this->getRequestParams()->getInt(RequestParams::PARAM_TYPE_BODY,
+            CommonParams::PARAMETER_EMP_NUMBER);
+        $this->getEmpEmergencyContactService()->deleteEmployeeEmergencyContacts($empNumber, $sequenceNumbers);
+        return new EndpointDeleteResult(ArrayModel::class, $sequenceNumbers);
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function getValidationRuleForDelete(): ParamRuleCollection
+    {
+        return new ParamRuleCollection(
+            new ParamRule(CommonParams::PARAMETER_EMP_NUMBER,
+                new Rule(Rules::IN_ACCESSIBLE_EMP_NUMBERS)
+            ),
+            new ParamRule(CommonParams::PARAMETER_IDS,
+                new Rule(Rules::ARRAY_TYPE)
+            ),
+        );
+    }
+
+    /**
+     * @inheritDoc
+     * @throws DaoException
+     * @throws RecordNotFoundException
+     */
+    public function getOne(): EndpointGetOneResult
+    {
+        $empNumber = $this->getRequestParams()->getInt(RequestParams::PARAM_TYPE_ATTRIBUTE,
+            CommonParams::PARAMETER_EMP_NUMBER);
+        $seqNo = $this->getRequestParams()->getInt(RequestParams::PARAM_TYPE_ATTRIBUTE, CommonParams::PARAMETER_ID);
+        $emergencyContact = $this->getEmpEmergencyContactService()->getEmployeeEmergencyContact($empNumber, $seqNo);
+        $this->throwRecordNotFoundExceptionIfNotExist($emergencyContact, EmpEmergencyContact::class);
+
+        if (!$emergencyContact instanceof EmpEmergencyContact) {
```

Comment:
remove this condition since you added `$this->throwRecordNotFoundExceptionIfNotExist($emergencyContact, EmpEmergencyContact::class);`